free-2.md

```diff
@@ -46,6 +61,8 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+There are several DSLs in this domain in the form of games. One such example is [Fiz](http://brewfiz.com/) in which you are in charge of running your own brewery. There are also other tools available online such as the Brewer's Friend [ABV Calculator](http://www.brewersfriend.com/abv-calculator/). This allows users to enter information about their brew and determine the alcohol-by-volume at that point in the process. I think that each of these examples has good qualities. The game is more user-friendly and serves a very different purpose than the calculator. I think that the calculator could be improved to have more functionality and that the game is not very helpful to someone looking to get quick information for brewing.
+
```

Comment:
The examples you provided were super cool! I didn't even know things like this existed. I imagine the DSL you might end up creating would look something in between the game and the calculator.